### ISSUE TITLE
Failed calibration - frequency not increasing each step

### ISSUE THREAD

**Issue by SinglerGold2** [bigtreetech/Eddy#79]
When I run `PROBE_EDDY_CURRENT_CALIBRATE_AUTO CHIP=btt_eddy` , I get the error Failed calibration - frequency not increasing each step . 
I  followed  the setup guide but I can't find a solution.

These are the results for the following commands:

`LDC_CALIBRATE_DRIVE_CURRENT CHIP=btt_eddy`
and then 
`PROBE_EDDY_CURRENT_CALIBRATE_AUTO CHIP=btt_eddy`

![image](https://github.com/user-attachments/assets/cba6a5c2-cf8a-41c3-ba78-0a38c95c19eb)

![image](https://github.com/user-attachments/assets/30c0fa7a-7a36-4b54-b206-540049853f07)

Device: BTT Eddy USB
Connected to a BTT Pad 7 with CM4

It worked for a few minutes after the first setup. Then I had to restart the Pad and now I have that issue

**Comment by SinglerGold2** [comment]
It worked after a reboot of the system... and stopped working again after a firmware reset after I changed some stuff in the printer.cfg which were not eddy related

**Comment by BTTJUSTIN** [comment]
Thanks for reaching out. Our support team does not operate using the GitHub platform but we would be happy to help you via email based ticketing system. Please visit https://biqu.equipment/pages/technical-support 
and submit a ticket and we will be in touch with you soon.

**Comment by SinglerGold2** [comment]
> Thanks for reaching out. Our support team does not operate using the GitHub platform but we would be happy to help you via email based ticketing system. Please visit https://biqu.equipment/pages/technical-support and submit a ticket and we will be in touch with you soon.

trying/waiting to get that  confirmation email from your system for hours now... nothing. And yes, I checked my junk box.

**Comment by kingjamez** [comment]
For future peeps finding this via google like I did:
My solution to the error "Frequency not increasing each step" was to ensure that the EDDY was 2-3mm below the zero of the nozzle. In my case it was probably 5mm above the nozzle.

I took a full size SD card and moved the eddy down until the SD card fit between the EDDY and the bed when the nozzle was at zero.
After that the calbiration worked immediately.
